### PR TITLE
OCP4: Fix instructions of rules that set kubelet related sysctls, use the sysctl probe

### PR DESCRIPTION
**Pull request by jhrozek**
#### Description:

- OCP4: Use the sysctl probe when checking kubelet sysctls
- OCPBUGS-11334: OCP4: Fix instructions for rules that set the kubelet related sysctls

#### Rationale:

The instructions were confusing and were instructing to check one particular file which doesn't work for cases where the sysctls are either remediated in other way or just are correct by default. Let's fix the text.

We used to check one particular file and were using the lineinfile probe. This would fail in case someone just uses a different file for setting the sysctls. Let's use the sysctl probe instead.

#### Review Hints:

- Create a remediation that sets the kubelet related sysctls, but with a different file name. The scan should PASS.
- inspect the instructions

### CODE REVIEW

**Comment by jhrozek** [comment]
@Vincent056 you need to click on the approve button in review, there's no prow in this repo

File: applications/openshift/kubelet/kubelet_enable_protect_kernel_sysctl_kernel_panic_on_oops/rule.yml [hunk 1]
```diff
@@ -92,7 +103,8 @@ references:
   srg: SRG-APP-000516-CTR-001325
 
 template:
-  name: lineinfile
-  vars:
-    path: /etc/sysctl.d/90-kubelet.conf
-    text: "kernel.panic_on_oops=1"
+    name: sysctl
```
Comment (rhmdnd) [inline applications/openshift/kubelet/kubelet_enable_protect_kernel_sysctl_kernel_panic_on_oops/rule.yml:106]: nit: doesn't look like this affects the tests, but the indentation shifted right a layer.

**Review by rhmdnd — APPROVED** [review]

**Pull request merged**